### DISCUSSION TITLE
GitHub Copilot extension certification

### DISCUSSION THREAD

**Discussion by jeffabailey** [intitni/CopilotForXcode#533] [Ideas]
Are there any plans to work with GitHub to certify this extension for use with GitHub Copilot?

In other words, working to get XCode added here?

https://docs.github.com/en/copilot/github-copilot-chat/copilot-chat-in-ides/about-github-copilot-chat-in-your-ide

**Comment by intitni** [comment]
The extensions/plugins for the apps in the list are all maintained by GitHub, I don't think they want any third party app not under their control on the list. They need to assure users that the apps they recommend is safe.

**Reply by jeffabailey** [reply]
They have a certification program. If you're interested in learning about it, I'll find a contact for you.

**Reply by martinm82** [reply]
It really would be helpful to get some "certified by github" badge on this extension to make it easier in big coorporations to get this plugin accepted.
